models/hh_psc_alpha_implicit.nestml

```diff
@@ -66,10 +66,10 @@ neuron hh_psc_alpha_implicit:
   equations:
     # synapses: alpha functions
     I_syn_ex'' = -I_syn_ex' / tau_syn_ex
-    I_syn_ex' = I_syn_ex' - ( I_syn_ex / tau_syn_ex ) * 1 [ms/s]
+    I_syn_ex' = I_syn_ex' - ( I_syn_ex / tau_syn_ex )
```

Comment:
[memo2] this is a highly disturbing way of writing this ODE, there has to be another way!